testing/src/test/java/io/stargate/it/http/graphql/graphfirst/SelectTest.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.it.http.graphql.graphfirst;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import com.jayway.jsonpath.JsonPath;
+import io.stargate.it.driver.CqlSessionExtension;
+import io.stargate.it.driver.TestKeyspace;
+import io.stargate.it.http.RestUtils;
+import io.stargate.it.storage.StargateConnectionInfo;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(CqlSessionExtension.class)
+public class SelectTest extends GraphqlFirstTestBase {
+  private static GraphqlFirstClient CLIENT;
+  private static String NAMESPACE;
+
+  @BeforeAll
+  public static void setup(StargateConnectionInfo cluster, @TestKeyspace CqlIdentifier keyspaceId) {
+    CLIENT =
+        new GraphqlFirstClient(
+            cluster.seedAddress(), RestUtils.getAuthToken(cluster.seedAddress()));
+    NAMESPACE = keyspaceId.asInternal();
+    CLIENT.deploySchema(
+        NAMESPACE,
+        "type Foo @cql_input {\n"
+            + "  pk1: Int! @cql_column(partitionKey: true)\n"
+            + "  pk2: Int! @cql_column(partitionKey: true)\n"
+            + "  cc1: Int! @cql_column(clusteringOrder: ASC)\n"
+            + "  cc2: Int! @cql_column(clusteringOrder: DESC)\n"
+            + "}\n"
+            + "\n"
+            + "type Query { fooPartial(pk1: Int!, pk2: Int!, cc1: Int!): [Foo]"
```

Comment:
because of the `QueryFatcher` change discussed here: https://github.com/stargate/stargate/pull/709#discussion_r582779608
both queries need to return `[Foo]` now.